src/FirmwarePlugin/APM/ArduSubFirmwarePlugin.cc

```diff
@@ -164,11 +164,6 @@ int ArduSubFirmwarePlugin::remapParamNameHigestMinorVersionNumber(int majorVersi
     return majorVersionNumber == 3 ? 6 : Vehicle::versionNotSetValue;
 }
 
-int ArduSubFirmwarePlugin::manualControlReservedButtonCount(void)
-{
-    return 0;
```

Comment:
This is what seems odd in previous code. I would have expected that if ArduSub wanted those button bits that it would have reserved them for use.

Reply:
I see it now. Not really because it used a jsScriptButton flag thing to turn off all of the normal button setting ui.

Which means I think:
* The reserved button thing can go away since ArduSub already has the other jsScript flag thing to signal usage.
* The button bits should be passed to MANUAL_CONTROL

Which I think means that this pull is correct to the above spec.